src/backport.ts

```diff
@@ -10,9 +10,19 @@ import {
   fetchPr,
 } from "./github.ts";
 
+if (
+  Deno.env.get("BACKPORTER_GITEA_FORK") === undefined ||
+  Deno.env.get("BACKPORTER_GITHUB_TOKEN") === undefined
```

Comment:
```suggestion
  !Deno.env.get("BACKPORTER_GITEA_FORK") ||
  !Deno.env.get("BACKPORTER_GITHUB_TOKEN")
```

Will also catch empty string.